### PR TITLE
EBODAC-638 - QA - After Enroll/Unenroll the screen comes back on firs…

### PR DESCRIPTION
**Pull request by MikolajBasSoldevelo**
…t page

### CODE REVIEW

**Comment by kkaczmarczyk** [comment]
test this please

**Comment by kkaczmarczyk** [comment]
test this please

File: ebodac/src/main/resources/webapp/js/controllers.js [hunk 1]
```diff
@@ -542,17 +546,21 @@
             $scope.lookupRefresh = !$scope.lookupRefresh;
         };
 
+        $scope.refreshGridAndStayOnSamePage = function() {
```
Comment (pmuchowski) [inline ebodac/src/main/resources/webapp/js/controllers.js]: on enrollment advanced you always have only one page, so this changes are not needed

File: ebodac/src/main/resources/webapp/js/directives.js [hunk 2]
```diff
@@ -354,6 +361,13 @@
                         }
                     }).trigger('reloadGrid');
                 });
+
```
Comment (pmuchowski) [inline ebodac/src/main/resources/webapp/js/directives.js]: remove changes from here also

**Pull request merged**